modules/artemis/example_test.go

```diff
@@ -0,0 +1,34 @@
+package artemis_test
+
```

Comment:
Oh super neat to not embed test code into the docs 👏👏👏

Reply:
🙂

Reply:
BTW I'm planning to have an example generated alongside the module, which would simplify any code snippet. Thanks for the idea! 💡